src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs

```diff
@@ -37,6 +37,9 @@ public class NewAutomationUpdateManagementAzureQuery : AzureAutomationBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
         public string[] Locaton { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
+        public string[] Location { get; set; }
```

Comment:
Shopuld use a LocationCompleter attribute for this

Reply:
Hi @markcowl , can you explain little bit about this? is there existing attribute with the Name "LocationCompleter" or should I create one? Is there example of this that has been used in other code?

Reply:
See: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/parameter-best-practices.md#location-completer

Reply:
@markcowl , Thank you so much. Updated